### PR TITLE
[WIP] Release 0.4.3: System Updates and Minor Bug Fixes

### PR DESCRIPTION
**Pull request by ax3l**
This release adds updates and new HPC system templates. Important bug fixes include I/O work-arounds for issues in OpenMPI 2.0-4.0 (mainly with HDF5), guards for particle creation with user-defined profiles, a fixed binomial current deposition, checks for the number of devices in grid distributions and container (Docker & Singularity) modernizations.

Thanks to Axel Huebl, Alexander Debus, Igor Andriyash, Marco Garten, Sergei Bastrakov, Adam Simpson, Richard Pausch, Juncheng E, Klaus Steiniger, and René Widera for contributions to this release!

## To Do

- [x] reviewers: please check the changelog for typos
- [ ] add JUWELS (FZJ) #2874
- [x] add @ejcjason's real name in changelog: Juncheng E

### CODE REVIEW

**Comment by PrometheusPi** [comment]
@ax3l: @ejcjason's real name is Juncheng E - see #2842

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,63 @@
...
+0.4.3
+-----
+
+**Date:** 2019-02-13
+
+System Updates and Minor Bug Fixes
+
+This release adds updates and new HPC system templates. Important bug
+fixes include I/O work-arounds for issues in OpenMPI 2.0-4.0 (mainly
+with with HDF5), guards for particle creation with user-defined
```
Comment (sbastrakov) [inline CHANGELOG.md]: Double `with`

**Comment by ax3l** [comment]
Huh, that update didn't work as expected... o.0

**Comment by ax3l** [comment]
Hm, I might have overdone the branch renaming and base switching.
Re-opened in #2886

**Pull request closed without merging**